torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -51,7 +51,7 @@ class FSDPCommContext:
 
     def lazy_init(self, device: torch.device):
         self.device_handle = _get_device_handle(device.type)
-        if device.type not in ["cuda", "hpu"]:
+        if device.type not in ["cuda", "hpu", torch._C_get_privateuse1_backend_name()]:
```

Comment:
Should `torch._C_get_privateuse1_backend_name` be
`torch._C._get_privateuse1_backend_name`?
(missing one `.`)